packages/frontend/src/components/UserCompletionModal/index.tsx

```diff
@@ -63,86 +69,108 @@ const UserCompletionModal: FC = () => {
     const isValidOrganizationDomain = !validateOrganizationEmailDomains([
         getEmailDomain(user.data?.email || ''),
     ]);
+
     return (
-        <Overlay
-            isOpen={!isSuccess}
-            enforceFocus
-            hasBackdrop
-            canEscapeKeyClose={false}
-            canOutsideClickClose={false}
-        >
-            <UserCompletionModalCard elevation={2}>
-                <Title>Nearly there...</Title>
-                <Subtitle>Tell us a bit more about yourself</Subtitle>
-                <Form
-                    name="complete_user"
-                    methods={methods}
-                    onSubmit={handleSubmit}
-                >
-                    {user.data.organizationName === '' && (
-                        <InputField
-                            label="Organization name"
-                            name="organizationName"
-                            placeholder="Enter company name"
-                            disabled={isLoading}
-                            rules={{
-                                required: 'Required field',
-                            }}
-                            defaultValue={user.data.organizationName}
-                        />
-                    )}
-                    <InputSelect
-                        label="What's your role?"
-                        name="jobTitle"
-                        disabled={isLoading}
-                        options={jobTitles}
-                        rules={{
-                            required: 'Required field',
-                        }}
-                    />
-                    {user.data.organizationName === '' &&
-                        isValidOrganizationDomain && (
-                            <Checkbox
-                                name="enableEmailDomainAccess"
+        <>
+            <Modal
+                opened={!isSuccess}
+                size="auto"
+                onClose={() => {}}
+                withCloseButton={false}
+                centered
+                title={<Title order={4}>Nearly there...</Title>}
+                styles={{
+                    title: {
+                        width: '100%',
+                        textAlign: 'center',
+                    },
+                    header: {
+                        padding: '1rem',
+                        paddingBottom: '0.75rem',
```

Comment:
is there a reason why you are using rems and not default sizing properties?

Reply:
great point, let me move the subheading inside the title